lms/static/sass/elements/_program-card.scss

```diff
@@ -89,8 +97,30 @@
   }
 
   .certificate-status {
-    font-size: em(12);
-    color: $gray;
+    .link {
+      font-size: em(12);
+      color: $gray;
+    }
+ 
+    .secondary {
+      @extend %hide-until-focus;
+    }
+
+    .link {
```

Comment:
NIT: I dont know for sure but I thought we had some link styling extends in _utilities.scss (or a similar partial)